x-pack/plugins/stack_connectors/server/connector_types/sentinelone/render.ts

```diff
@@ -9,13 +9,15 @@ import { map } from 'lodash';
 import { set } from '@kbn/safer-lodash-set/fp';
 import type { EcsSecurityExtension as Ecs } from '@kbn/securitysolution-ecs';
 import { ExecutorParams } from '@kbn/actions-plugin/server/sub_action_framework/types';
+import { Logger } from '@kbn/core/server';
 import { SUB_ACTION } from '../../../common/sentinelone/constants';
 
 interface Context {
   alerts: Ecs[];
 }
 
 export const renderParameterTemplates = (
+  logger: Logger,
```

Comment:
The logger param is not used in the function here. Can we do a better type for where Logger is required so that you don't have to pass params into functions where it is unused.